setup.cfg

```
@@ -74,7 +74,7 @@ filterwarnings =
     # Ignore FutureWarning for change in group_keys behavior: no changes required on the dask side.
     # https://pandas.pydata.org/docs/dev/whatsnew/v1.5.0.html#using-group-keys-with-transformers-in-groupby-apply
     ignore:Not prepending group keys:FutureWarning
-    ignore:.*:sqlalchemy.exc.RemovedIn20Warning
+    ignore:.*::sqlalchemy.exc.RemovedIn20Warning
```

the syntax is `action:message:category:module:line` and so this ignores all warnings of any type from the module `sqlalchemy.exc.RemovedIn20Warning`?

---

Apologies, I based this off a similar change made to Distributed's setup.cfg, which I assume is also incorrect for the same reason:

https://github.com/dask/distributed/blob/7514fa3d1c0051e75632b5a30aef2ef420cbbf23/setup.cfg#L83

In that case, unless there are other `sqlalchemy` warnings we are expecting and want to ignore, would it make sense to instead use a filter like

```suggestion
    ignore:.*These feature\(s\) are not compatible with SQLAlchemy 2\.0\..*:
```